docs/smart-contracts/getting-started/create-an-app.mdx

```diff
@@ -425,6 +425,22 @@ If you're still having problems, join our Discord (link above) or [open an issue
 
 This means that Testnet is down, and you probably just need to wait a while and try again.
 
+### Freigther popup mentioning SSL required
+
+- Install a vite plugin: `npm install --save @vitejs/plugin-basic-ssl`
+- Edit `astro.config.mjs`:
+
+```mjs
+import { defineConfig } from "astro/config";
+import basicSsl from "@vitejs/plugin-basic-ssl";
+
+export default defineConfig({
+  vite: { plugins: [basicSsl()] },
+});
+```
+
+- Open `https://localhost:4321`
```

Comment:
This is an interesting, and valid I would say, approach to troubleshooting this problem. But, there's also a (perhaps simpler) way to get Freighter to behave with non-HTTPS sites. When configuring a network in Freighter there's a checkbox which will allow connecting to insecure servers.

I'd love to see that option alongside this approach to it.

Reply:
I didn't realize that thanks